res/css/views/rooms/_IRCLayout.scss

```diff
@@ -15,115 +15,141 @@ limitations under the License.
+                    height: $font-14px !important;
+                    width: $font-14px !important;
+                    font-size: $font-10px !important;
+                    line-height: $font-15px !important;
+                }
+            }
+
+            .mx_EventTile_e2eIcon {
+                position: absolute;
+                right: unset;
+                left: unset;
+                top: 0;
+
+                padding: 0;
+
+                flex-shrink: 0;
+                flex-grow: 0;
+
+                height: $font-18px;
+
+                background-position: center;
             }
         }
 
-        .mx_EventTile_line, .mx_EventTile_reply {
+        .mx_EventTile_line,
+        .mx_EventTile_reply {
+            order: 3;
             padding: 0;
             display: flex;
             flex-direction: column;
-            order: 3;
             flex-grow: 1;
             flex-shrink: 1;
             min-width: 0;
         }
 
-        > .mx_EventTile_avatar {
-            order: 1;
-            position: relative;
-            top: 0;
-            left: 0;
-            flex-shrink: 0;
-            height: $irc-line-height;
-            display: flex;
-            align-items: center;
-
-            // Need to use important to override the js provided height and width values.
-            > .mx_BaseAvatar, > .mx_BaseAvatar > * {
-                height: $font-14px !important;
-                width: $font-14px !important;
-                font-size: $font-10px !important;
-                line-height: $font-15px !important;
-            }
-        }
-
         .mx_MessageTimestamp {
             font-size: $font-10px;
             text-align: right;
         }
 
-        > .mx_EventTile_e2eIcon {
-            position: absolute;
-            right: unset;
-            left: unset;
-            top: 0;
-
-            padding: 0;
-
-            flex-shrink: 0;
-            flex-grow: 0;
-
-            height: $font-18px;
-
-            background-position: center;
-        }
-
         .mx_EventTile_line {
             .mx_EventTile_e2eIcon,
             .mx_TextualEvent,
             .mx_MTextBody {
                 display: inline-block;
                 // add a 1px padding top and bottom because our larger emoji font otherwise gets cropped by anti-zalgo
-                padding: 1px 0;
+                padding: var(--IRCLayout_EventTile_line_child-padding-block) 0;
             }
         }
 
         .mx_EventTile_reply {
             order: 4;
         }
 
+        .mx_EventTile_msgOption {
+            order: 5;
+            flex-shrink: 0;
+
+            .mx_ReadReceiptGroup {
+                top: -0.3rem; // ($irc-line-height - avatar height) / 2
+            }
+        }
+
         .mx_EditMessageComposer_buttons {
             position: relative;
         }
 
+        .mx_ThreadSummary,
         .mx_ReactionsRow {
-            padding-left: 0;
-            padding-right: 0;
+            margin-inline: 0;
```

Comment:
This works but it is redundant- removing the inline margin declaration from `_EventTile.scss` looks straightforward.